bucket/jdtls.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "1.10.0-202204081744",
+    "description": "Java language server",
+    "homepage": "https://projects.eclipse.org/projects/eclipse.jdt.ls",
+    "license": "EPL v2.0",
+    "suggest": {
+        "JDK": "java/openjdk",
+        "Python": "python"
+    },
+    "url": "https://download.eclipse.org/jdtls/snapshots/jdt-language-server-1.10.0-202204081744.tar.gz",
+    "hash": "da4c832d89e1e66932eb809c5760043fe1abdbefc1a4492a8022edd354c873c5",
+    "installer": {
+        "script": "shim \"$(currentdir python $global)\\python.exe\" $global jdtls \"$dir\\bin\\jdtls\""
```

```suggestion
        "script": "shim (search_in_path python) $global jdtls \"$dir\\bin\\jdtls\""
```

A more robust way would be this

---

Yeap, much cleaner thanks! Had to change `(search_in_path python)` to `(search_in_path python.exe)` for this to work correctly!